pages/quiz.tsx

```diff
@@ -1,17 +1,27 @@
 import Link from 'next/link'
 import Router from 'next/router'
-import React, { FC, MouseEvent, useState } from 'react'
 
-import FoodQuestion from '../components/FoodQuestion'
+import { FC, MouseEvent, useState, useEffect, useContext } from 'react'
+
 import Nav from '../components/Nav'
+import FoodQuestion from '../components/FoodQuestion'
 import TimeQuestion from '../components/TimeQuestion'
+
 import data from '../utils/data.json'
 import { formatQuery } from '../utils/helpers'
 
+import { SelectedIngredientsContext } from '../context/state'
+
 const Quiz: FC = () => {
   const [time, setTime] = useState('')
   const [ingredients, setIngredients] = useState([])
 
+  const { userIngredients, setUserIngredients } = useContext(SelectedIngredientsContext)
+
+  useEffect(() => {
```

Comment:
Is this needed? When we call `setUserIngredients` it should update ingredients

Reply:
It do be working though.